scripts/build-docker.sh

```diff
@@ -15,7 +15,9 @@ VERSION="${VECTOR_VERSION:-"$(cargo vdev version)"}"
 DATE="${DATE:-"$(date -u +%Y-%m-%d)"}"
 PLATFORM="${PLATFORM:-}"
 PUSH="${PUSH:-"true"}"
-REPO="${REPO:-"timberio/vector"}"
+# TODO revert this when ready to merge
+#REPO="${REPO:-"timberio/vector"}"
```

Comment:
Still TODO?

Reply:
It'll be the last thing I revert back as mentioned in the opening comment.